pkg/apis/enmasse/enmasse/v1beta1/types.go

```diff
@@ -78,9 +78,9 @@ type CertificateSpec struct {
 }
 
 type ExposeSpec struct {
-	Type                string `json:"route"`
-	RouteServicePort    string `json:"routeServicePort"`
-	RouteTlsTermination string `json:"routeTlsTermination"`
+	Type               string `json:"route"`
+	RouteServicePort   string `json:"routeServicePort"`
+	RouteTLSermination string `json:"routeTlsTermination"`
```

Comment:
I think we should leave these ones alone for now actually (the files in `pkg/apis/*`), since these shouldn't even be here (and when we depend on them properly, we'd need to change some stuff back).

For example, see:
https://issues.redhat.com/browse/INTLY-4642
https://issues.redhat.com/browse/INTLY-4644